pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -378,7 +377,7 @@ func (b *volumeBinder) AssumePodVolumes(assumedPod *v1.Pod, nodeName string, pod
 	klog.V(4).Infof("AssumePodVolumes for pod %q, node %q", podName, nodeName)
 	start := time.Now()
 	defer func() {
-		metrics.VolumeSchedulingStageLatency.WithLabelValues("assume").Observe(time.Since(start).Seconds())
+		metrics..WithLabelValues("assume").Observe(time.Since(start).Seconds())
```

Comment:
`metrics..WithLabelValues` is illegal.

Reply:
omg, wrong delete shortcut :yum: thanks ^